### DISCUSSION TITLE
Auf der 7530AX gibt es kein loop ihm make menuconfig

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#661] [Allgemein]
Wer es nicht glauben will. Hier mal ein Video egal welche FW nutzt jedes Mal das gleiche Ergebnis 
[Bildschirmaufzeichnung vom 06.12.2022, 17:07:15.webm](https://user-images.githubusercontent.com/79326105/205963045-aa452246-60fb-4f71-9a65-9345b31219a8.webm)

**Comment by fda77** [comment]
Und? Ist das ne Frage? Oder eine Problem?
Kernelmodule müssen ja nicht unbedingt eine .ko Datei sein sondern können auch direkt in den Kernel compiliert sein

**Comment by freetz-ng-mod** [comment]
Ist das eine Antwort, nein. Aber lass gut sein. Ich habe ja alles wieder drin. Nur die anderen dann halt nicht
![@7530ax-client – System](https://user-images.githubusercontent.com/79326105/205971629-bd7d1366-7e02-482a-ad1a-1d7fc9b27e75.png)

**Comment by MasterRoCcO** [comment]
Er hat aber recht @fda77 nur so zur Information! Habe extra noch mal ein neues Image gebaut.
Das alte Image
![7530ax_alte_Image](https://user-images.githubusercontent.com/47879729/205977592-dd03289e-8319-4edf-bebe-51f5c2253954.png)
Das neue
![7530ax_neue_Image](https://user-images.githubusercontent.com/47879729/205977730-f4175a29-dca8-46c1-acab-177f1ed7bb54.png)

Und wie du sehen kannst, wird nichts angezeigt

**Comment by fda77** [comment]
Selbstgespäche bitte sonstwo führen
